node/resolvers/catalog/index.ts

```diff
@@ -105,16 +127,21 @@ const brandFromList = async (
 const getBrandId = async (
   brand: string | undefined,
   catalog: Context['clients']['catalog'],
-  isVtex: boolean
+  isVtex: boolean,
+  logger: Context['clients']['logger']
 ) => {
   if (!brand) {
     return null
   }
   if (!isVtex) {
     return brandFromList(brand, catalog)
   }
-  const brandPagetype = await catalog.pageType(brand)
-  if (brandPagetype.pageType !== 'Brand') {
+  const slugfied = catalogSlugify(brand)
+  const brandPagetype = await catalog.pageType(slugfied).catch(() => null)
```

Comment:
```suggestion
  const brandPagetype = await catalog.pageType(slugified).catch(() => null)
```

Reply:
maybe you should handle and log the rejection (up to you, might not make sense in this case)